### ISSUE TITLE
Length ranges in symmetry modes

### ISSUE THREAD

**Issue by ali-j-scott** [RosettaCommons/RFdiffusion#47]
Hi,

I've noticed that when using a symmetric mode (cyclic, dihedral, etc), it's not possible to supply a length range for the new diffused region. A single value must be used, otherwise 'ValueError: Sequence length must be divisble by n' is returned.

I'm guessing it's because the lengths of the diffused regions in each chain/monomer aren't tied, so usually the total sequence length ends up indivisible by the oligomeric state? 

Is this something that might be possible in future updates?

Thanks!

Ali

**Comment by broomsday** [comment]
Yes, I encountered this issue also and had to workaround by doing multiple runs with each different length specified.
